src/vs/platform/terminal/common/terminal.ts

```diff
@@ -4,9 +4,171 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Event } from 'vs/base/common/event';
+import { IDisposable } from 'vs/base/common/lifecycle';
 import { IProcessEnvironment } from 'vs/base/common/platform';
 import { URI } from 'vs/base/common/uri';
 
+export interface IRemoteTerminalAttachTarget {
+	id: number;
+	pid: number;
+	title: string;
+	cwd: string;
+	workspaceId: string;
+	workspaceName: string;
+	isOrphan: boolean;
+}
```

Comment:
Think this can be removed for now, we'll consolidate and hopefully share the interface between local and remote when we get to https://github.com/microsoft/vscode/issues/116467